test/controllers/visit.controller.test.js

```diff
@@ -0,0 +1,131 @@
+import mongoose from 'mongoose';
+import { expect, request, useDatabase } from '../config';
+import User from '../../server/models/user.model';
+import VisitFactory from '../factories/visit.factory';
+import PropertyFactory from '../factories/property.factory';
+import UserFactory from '../factories/user.factory';
+import { addUser } from '../../server/services/user.service';
+import { addProperty } from '../../server/services/property.service';
+
+useDatabase();
+
+let userToken;
+
+const userId = mongoose.Types.ObjectId();
+const systemUser = UserFactory.build({ _id: userId, role: 0, activated: true });
+
+beforeEach(async () => {
+  userToken = await addUser(systemUser);
```

Comment:
This should be adminToken. It is better to properly name the different variables to ensure that they are not confusing when introducing new tests later.
```suggestion
  adminToken = await addUser(systemUser);
```